src/BlockEntities/PistonEntity.cpp

```diff
@@ -0,0 +1,52 @@
+// PistonEntity.cpp
+
+// Implements the cPistonEntity class representing a single piston in the world
+
+#include "Globals.h"  // NOTE: MSVC stupidness requires this to be the same across all modules
+#include "PistonEntity.h"
+
+cPistonEntity::cPistonEntity(BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta, Vector3i a_Pos, cWorld * a_World):
```

Comment:
For some reason, the constructor seems as if it never gets called, even though https://github.com/cuberite/cuberite/blob/1590d5bd1243f6406387bf8c1d319593ffe83983/src/BlockEntities/BlockEntity.cpp#L108-L109 has been added